pkgs/development/python-modules/steamfiles/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  protobuf,
+  protobuf3-to-dict,
+}:
+buildPythonPackage {
+  pname = "steamfiles";
+  version = "0-unstable-2020-10-26";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "twstagg";
+    repo = "steamfiles";
+    rev = "18bb08d5ff8b0e36d4cc14453deaf58d721339e4";
+    hash = "sha256-CRzX4mKvBVYC0ayu2bq6aAzhAWPW0m737cw1gfBRj7U=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    protobuf
+    protobuf3-to-dict
+  ];
+
```

Comment:
Can we execute the tests https://github.com/twstagg/steamfiles/tree/master/tests ?